docs/api.md

```diff
@@ -664,6 +664,32 @@ from. Optionally a single string joining container id's with commas
 None
 ```
 
+## stats
+
+The Docker API parallel to the `docker stats` command.
+This will stream statistics for a specific container.
+
+`aggregate`, if 0 will return an iterable stream object of messages.
```

Comment:
I believe this is mechanism for "batching" stats. 
But I think it's a bit problematic since it's blocking and can actually get deadlock if stats stream stopped?